### DISCUSSION TITLE
How to override font globally without using vfonts?

### DISCUSSION THREAD

**Discussion by oivvio** [tusen-ai/naive-ui#2887] [Q&A]
How do I override fonts globally without using vfonts?

The documentation says this:

> If you don't use `vfont` and want change global fonts by customzing theme, you need to use `n-global-style` to achieve that. Components' font won't be responsive to `theme-overrides` without `n-global-style`.

But does not give an example about how to override the font with `n-global-style`. A basic example would be much appreciated. Thanks in advance.

**Comment by 07akioni** [comment]
Currently you can write your custom css.

```
body {
  font-family: xxx
}
```

**Reply by NightGreyMoon** [reply]
But font will be overridden by components like n-table

**Comment by viktorsheep** [comment]
It would be really helpful if they could provide a simple example for this. I've been trying to achieve this for some times now.. T_T

**Comment by jahnli** [comment]
This issue does not have any recent activity. If you are still experiencing similar problems, open a new error, including a minimal copy of the problem
